### ISSUE TITLE
If user enters incorrect password they see Authentication Required popup

### ISSUE THREAD

**Issue by hunt3ri** [thinkWhere/DMIS#100]
If users enters an incorrect password when logging in they see the standard browser Authentication Required popup instead of an error message.

**Comment by popelnuh** [comment]
The issue is now fixed.

**Comment by LindaAlblas** [comment]
@popelnuh I don't think this issue is fixed. Is it only happening on certain browsers? It is still a bug when using Chrome on my machine.

**Comment by LindaAlblas** [comment]
@hunt3ri could this be because the staging site is not using HTTPS (yet)? I don't get the standard browser Authentication Required popup when running the app locally.

**Comment by hunt3ri** [comment]
@LindaAlblas I meant to let you know that I managed to replicate this locally.  Serving the application directly via Python rather than thru Angular causes the 401 popup to appear.  We can look at this tomorrow.

**Comment by LindaAlblas** [comment]
@hunt3ri Ah ok. Thanks for testing.

**Comment by LindaAlblas** [comment]
It looks like Chrome catches the Basic 401 response before it gets to Angular. 

https://stackoverflow.com/questions/24130308/preventing-http-basic-auth-dialog-using-angularjs-interceptors
https://github.com/flask-restful/flask-restful/issues/172
